pkg/scheduler/metrics/metrics.go

```diff
@@ -23,90 +23,97 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-const schedulerSubsystem = "scheduler"
+const (
+	// SchedulerSubsystem - subsystem name used by scheduler
+	SchedulerSubsystem = "scheduler"
```

Comment:
Renaming metrics is a breaking change. It is not backward and is going to break monitoring systems that rely on these metrics. This PR should be reverted.

Reply:
Yeah, that's a valid concern and also the option I was initially suggesting in https://github.com/kubernetes/kubernetes/issues/63493#issuecomment-389904406. I've cc'd there sig-scheduling and you - following lazy consensus since there weren't any objections :)

So, what do you think about re-adding the old metric alongside the new one (instead of replacing it)?

Reply:
Wasn't this metric broken anyway?

Reply:
I don't think so.. IIUC we were just not capturing the values properly in our test framework (which expects summary instead of histogram).

Reply:
If we're going to add "aliases" for metrics, I think we should make more of a push toward prometheus best practices. The various histograms in this file and their relationships are a bit confusing.

Reply:
I agree with @misterikkit. Some of these names were not following Prometheus best practices. The new ones introduces in this PR are not following those best practices either. We should definitely keep the old ones for backward compatibility and maybe add new aliases based on the best practices.